tests/monitoring/monitoring_service/test_handlers.py

```diff
@@ -85,12 +86,14 @@ def setup_state_with_open_channel(context: Context) -> Context:
         channel_identifier=DEFAULT_CHANNEL_IDENTIFIER,
         participant1=DEFAULT_PARTICIPANT1,
         participant2=DEFAULT_PARTICIPANT2,
-        settle_timeout=DEFAULT_SETTLE_TIMEOUT,
         block_number=BlockNumber(42),
     )
     assert context.database.channel_count() == 0
 
     channel_opened_event_handler(event, context)
+    context.web3.eth.get_block = lambda x: Mock(
+        timestamp=context.web3.eth.block_number * 15 if x == "latest" else x * 15
+    )
```

Comment:
We might miss some edge cases during testing when time and block number always progress proportionally. But I don't have a good suggestion, since most changes would make the overall setup more complicated.